index.js

```diff
@@ -1,4 +1,4 @@
 const server = require('./api/server.js');
 
-const port = process.env.PORT || 6000;
+const port = process.env.PORT || 4000;
```

Comment:
Now your console.log down below is telling you your port is '6k' when it could be whatever the port is in your `.env` OR 4000. Again, this is exactly why you want the port variable in your log!